Doc/whatsnew/3.9.rst

```diff
@@ -105,6 +105,13 @@ Merge (``|``) and update (``|=``) operators have been added to the built-in
 :class:`dict` class.  See :pep:`584` for a full description.
 (Contributed by Brandt Bucher in :issue:`36144`.)
 
+removeprefix and removesuffix methods for strings
+-------------------------------------------------
+``str.removeprefix(prefix)`` and ``str.removesuffix(suffix)`` have been added to
+easily remove an unneeded prefix or a suffix from a string. Corresponding
+``bytes`` and ``bytearray`` methods have also been added.
+(Contributed by Dennis Sweeney in :issue:`18939`.)
+
```

Comment:
You should mention the PEP 616 somewhere.

For example: `See :pep:`616` for a full description.`